backend/src/packages/orders/libs/helpers/combine-filters.ts

```diff
@@ -0,0 +1,21 @@
+import { type SQL, and, eq } from 'drizzle-orm';
+import { type PgColumn } from 'drizzle-orm/pg-core';
+
+import { type DatabaseSchema } from '~/libs/packages/database/schema/schema.js';
+
+type Table = DatabaseSchema[keyof DatabaseSchema];
+
+const combineFilters = <TSchema extends Table>(
```

Comment:
I'm not sure about it for now because it's not fully customizable because in line `searchBy: Record<PgColumn['name'], number | null>,`  where `number | null` we should list all posible types of PgColums. May be if refactoring, we can think about it